tests/unit/database_tests.py

```diff
@@ -442,6 +442,17 @@ def test_all_docs_get(self):
         self.assertEqual(data['rows'][1]['key'], 'julia011')
         self.assertEqual(data['rows'][2]['key'], 'julia012')
 
+    def test_all_docs_get_with_long_type(self):
+        """
+        Test the all_docs GET request functionality
+        """
+        self.populate_db_with_documents()
+        data = self.db.all_docs(limit=10000000000, skip=10)
```

Comment:
When I check the type of `10000000000`  in the interpreter it is `int` rather than long`. 

```bash
~> python 
Python 2.7.10 (default, Jul 30 2016, 19:40:32) 
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> o = 10000000000
>>> type(o)
<type 'int'>
>>> exit()
```

I think we need to suffix `10000000000` with `L` so we have `10000000000L`